README.md

```diff
@@ -1,3 +1,30 @@
 # deal-observer
 
 A near-real-time database of active FIL+ deals
+
+### [api/](./api/)
+
+Public REST API exposing info about FIL+ deals.
+
+### [backend/](./backend/)
```

Comment:
"backend" is slightly confusing, as it's not clear where the api ends and the backend begins. Would "observer" or "ingestion" be clearer?

Reply:
We use the name "indexer" in piece-indexer and "observer" in "spark-observer". I find it weird to have "spark-observer-observer" component, that's why I used "deal-observer-backend" here.

I see how "backend" may be slightly confusing, though.

Can we find a better name conveying the information that this is the backend (hidden) service observing the deals in the background and updating the database?

Reply:
We also have `api` in `spark-api` though, do you find it confusing there?